azure-mgmt-resource/README.rst

```diff
@@ -54,25 +54,94 @@ future versions.
 Usage
 =====
 
-Examples
---------
+Authentication
+--------------
 
-We'll work on adding some documentation here, but for now, see the following
-for examples on using the ARM APIs.
+Authentication with Azure Resource Manager is done via tokens.
 
--  `Azure Resource Viewer Web Application Sample <https://github.com/Azure/azure-sdk-for-python/tree/master/examples/AzureResourceViewer>`__
--  `Azure Resource Manager Unit tests <https://github.com/Azure/azure-sdk-for-python/tree/master/azure-mgmt/tests>`__
+First, create a service principal using the instructions found
+`here <https://azure.microsoft.com/en-us/documentation/articles/resource-group-create-service-principal-portal/>`__.
 
-Authentication
---------------
+Then, use the following code to obtain an authentication token.
+
+.. code:: python
+
+    import requests
+
+    def get_token_from_client_credentials(endpoint, client_id, client_secret):
+        payload = {
+            'grant_type': 'client_credentials',
+            'client_id': client_id,
+            'client_secret': client_secret,
+            'resource': 'https://management.core.windows.net/',
+        }
+        response = requests.post(endpoint, data=payload).json()
+        return response['access_token']
+
+    # TODO: Replace endpoint, client id and secret for your application
```

Comment:
This works.  We could also pull them out as Constants in here to make it a bit more clear what you are editing.  Maybe leave the todo on the comments?  This is what I tend to do just so the stuff the user is supposed to be editing is all at the top of the code sample.